dape.el

```diff
@@ -1041,7 +1045,7 @@ Note requires `dape--source-ensure' if source is by reference."
     (dape-command-cwd)))
 
 (defun dape-config-autoport (config)
-  "Replace :autoport in CONFIG keys `command-args' and `port'.
+  "Replace :autoport in CONFIG keys `command-args', `command-env' and `port'.
```

Comment:
I am interested what the use case is here?

I would like some documentation here that both keys and values are replaced in `command-env` as that seams like a thing that could be confusing. (Or if we want the change the logic to only values that seams fine as well)

Reply:
I agree that modifying the keys in `command-env` could be unexpected, so went with your suggestion to replace `:autoport` in the values of `command-env` only.

Also, unrelated to my PR, I noticed that the existing docstring doesn't quite match the behavior of the function. It says/said:

> If `port' is not `:autoport' return config as is.

However, occurrences of `:autoport` in `command-args` are replaced with the value of the config key `port`, even if that was not `:autoport`. I tried to set this straight in the updated docstring.